src/modules/application/components/HomePage/HomePage.jsx

```diff
@@ -1,7 +1,13 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { Button } from "antd";
 
-const HomePage = ({ styles }) => <>HomePage</>;
+const HomePage = ({ styles }) => (
```

Comment:
I would encourage you to use our `useFelaEnhanced` (would send you a gitter) instead of `connectFela`. We are trying to omit any HOC usage :). The other thing is that it passes down all the `styles`, `rules` and `theme` props you might not even need to use and you don't have any overview of props it consumes for the styling because not all of them might be needed in the wrapped component.

Reply:
Ok, no need for a gist, it is being added to our CRA, so here is a PR, but typed, so you might want to omit those :D https://github.com/AckeeCZ/create-react-app/pull/31/files